CLI tools/CurseForge to Packwiz-Modrinth.py

```diff
@@ -131,30 +134,38 @@ def main() -> int:
         cmd = f'mmc-export -i "{mmc_zip_path}" -f Modrinth --modrinth-search loose -o "{mmc_zip_root}" -c "{modrinth_config}" -v {pack_version} --scheme {"{name}-{version}"}'
         os.system(cmd)
 
-        if is_legacy == False:
-            extract_file(mmc_zip_root + "\\Fabulously Optimized-" + pack_version + ".mrpack", "modrinth.index.json", git_path + "\\" + "Modrinth", "Modrinth manifest", "Git")
+        if not is_legacy:
+            extract_file(
+                mmc_zip_root + "\\Fabulously Optimized-" + pack_version + ".mrpack",
+                "modrinth.index.json",
+                git_path + "\\" + "Modrinth",
+                "Modrinth manifest",
+                "Git",
+            )
 
     # Export Modrinth pack and manifest via packwiz method
     if packwiz_modrinth_export:
         os.system(packwiz_exe_path + " modrinth export")
         for pack in os.listdir(packwiz_path):
-            if pack.endswith('.mrpack'):
-                if is_legacy == False:
-                    extract_file(packwiz_path + "\\" + pack, "modrinth.index.json", git_path + "\\" + "Modrinth", "Modrinth manifest", "Git")
+            if pack.endswith(".mrpack"):
+                if not is_legacy:
+                    extract_file(
+                        packwiz_path + "\\" + pack,
+                        "modrinth.index.json",
+                        git_path + "\\" + "Modrinth",
+                        "Modrinth manifest",
+                        "Git",
+                    )
                 os.replace(packwiz_path + "\\" + pack, os.path.expanduser("~/Desktop") + "\\" + pack)
-                print("Moved " + pack + " to desktop")
+                print(f"Moved {pack} to desktop")
         os.system(packwiz_exe_path + " refresh")
-    
-    if not refresh_only:
-        mmc_zip_root = str(Path(cf_zip_path).parents[0])
-        mmc_zip_path = mmc_zip_root + "\\Fabulously Optimized " + pack_version + ".zip"
-        #remove_mod_from_archive("Sodium", mmc_zip_path)
-        #remove_mod_from_archive("Iris", mmc_zip_path)
 
     return 0

```

Comment:
Is this actually used anywhere?

Reply:
@osfanbuff63 I guess not. I removed all these lines since `mmc_zip_root` and `mmc_zip_path` are just created and not used. If they are -- it'd be better to just comment lines from 148 to 152 ofc.

Reply:
I was actually talking about the `return 0` on line 154/163

Reply:
@osfanbuff63 oh, you're about `return 0`. Since it's default return code and nothing else can be explicitly returned by `main` -- just removed it as well.